configuration/persistence.md

```diff
@@ -232,44 +232,77 @@ Example:
+| `<item>.countStateChangesSince(ZonedDateTime)`                  | Gets the number of changes in persisted States of an Item since a certain point in time                                                                                         |
+| `<item>.countStateChangesUntil(ZonedDateTime)`                  | Gets the number changes in of persisted States of an Item until a certain point in time                                                                                         |
+| `<item>.countStateChangesBetween(ZonedDateTime, ZonedDateTime)` | Gets the number of changes in persisted States of an Item between certain points in time                                                                                        |
+| `<item>.getAllStatesSince(ZonedDateTime)`                       | Gets all persisted State changes for an Item since a certain point in time (returns Iterable<HistoricItem>)                                                                     |
+| `<item>.getAllStatesUntil(ZonedDateTime)`                       | Gets all persisted State changes for an Item until a certain point in time (returns Iterable<HistoricItem>)                                                                     |
+| `<item>.getAllStatesBetween(ZonedDateTime, ZonedDateTime)`      | Gets all persisted State changes for an Item between certain points in time (returns Iterable<HistoricItem>)                                                                    |
+| `<item>.removeAllStatesSince(ZonedDateTime)`                    | Removes all persisted States since a certain point in time                                                                                                                      |
+| `<item>.removeAllStatesUntil(ZonedDateTime)`                    | Removes all future persisted States for an Item until a certain point in time                                                                                                   |
+| `<item>.removeAllStatesBetween(ZonedDateTime, ZonedDateTime)`   | Removes all persisted States for an Item between certain points in time                                                                                                         |
 
 These extensions use the [default persistence service](#default-persistence-service).
 You may specify a different persistence service by appending a String as an optional additional parameter at the end of the extension.
 
+Some persistence services may not support persisting to other points in time than the current time and removing persisted States, therefore not all actions may be supported for all persistence services.
+
+Some extensions return a HistoricItem object.
+It represents the state of a persisted item at a certain point in time.
+The most useful methods of the HistoricItem object returned by some queries, are `.state` and `.getTimestamp`
```

Comment:
```suggestion
The most useful methods of the HistoricItem object returned by some queries, are `.getState()` and `.getTimestamp()`.
```

Reply:
That was the original text, and would work in DSL I believe. I changed it anyway.

Reply:
.state and .timestamp should theoretically also work in JS, because in Nashorn compatibility mode of GraalJS access to the properties is possible that way. I guess it’s the same for DSL — but I personally think that using the getters is better style 👍